src/date.rs

```diff
@@ -4,7 +4,7 @@
 //! ISO 8601 calendar date with time zone.
 #![allow(deprecated)]
 
-#[cfg(any(feature = "alloc", feature = "std", test))]
+#[cfg(any(feature = "alloc", feature = "std"))]
```

Comment:
The commit message on this one seems confusing -- AFAICT it was already only available on `alloc`/`std`, but now is no longer available on `test`?